addons/web/static/src/core/utils/html.js

```diff
@@ -19,16 +28,6 @@ export function createElementWithContent(elementName, content) {
     return element;
 }
 
-/**
- * Escapes content for HTML. Content is unchanged if it is already a Markup.
- *
- * @param {string|ReturnType<markup>} content
- * @returns {ReturnType<markup>}
- */
-export function htmlEscape(content) {
-    return content instanceof Markup ? content : markup(escape(content));
-}
-
```

Comment:
Probably best to export OWL's `htmlEscape` in this file for now and we remove it in master after freeze.